// Created on Feb 5.

syntax = "proto3";

package afs;

// Interface exported by the server.
service FileServer {
  // Open resp
  rpc Open(OpenReq) returns (OpenResp) {}
  
 /* // 
  rpc Read(ReadReq) returns (ReadResp) {}

 // 
  rpc Write(ReadReq) returns (ReadResp) {}

 // 
  rpc Close(ReadReq) returns (ReadResp) {}

 // 
  rpc Delete(ReadReq) returns (ReadResp) {}

 // 
  rpc GetAttr(ReadReq) returns (ReadResp) {}
*/

}

message OpenReq {
  string path = 1;
}

message OpenResp {
  int32 err = 1;
}
